cargo-shuttle/src/args.rs

```diff
@@ -46,25 +46,22 @@ pub struct ProjectArgs {
     pub name: Option<ProjectName>,
 }
 
-fn parse_working_directory(working_directory: &OsStr) -> Result<PathBuf, OsString> {
-    canonicalize(working_directory)
-        .map_err(|e| format!("could not turn {working_directory:?} into a real path: {e}").into())
-}
-
 #[derive(StructOpt)]
 pub enum Command {
+    #[structopt(about = "create user credentials for the shuttle platform")]
+    Auth(AuthArgs),
     #[structopt(about = "deploy a shuttle project")]
     Deploy(DeployArgs),
-    #[structopt(about = "view the status of a shuttle project")]
-    Status,
-    #[structopt(about = "view the logs of a shuttle project")]
-    Logs,
     #[structopt(about = "delete the latest deployment for a shuttle project")]
     Delete,
-    #[structopt(about = "create user credentials for the shuttle platform")]
-    Auth(AuthArgs),
+    #[structopt(about = "create a new shuttle project in an existing directory")]
```

Comment:
```suggestion
    #[structopt(about = "create a new shuttle project")]
```
From the PR description I understand it will create the directory when it does not exist yet :sweat_smile: